frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/state-update/storage-tab-state-update.ts

```diff
@@ -97,10 +97,14 @@ export const internalStorageDiskBusUpdater = ({
         VMWizardStorageType.WINDOWS_GUEST_TOOLS,
       ].includes(type)
     ) {
-      const resultValidation = newValidations.validateBus(new DiskWrapper(disk).getDiskBus());
+      const diskWrapper = new DiskWrapper(disk, true);
+      const resultValidation = newValidations.validateBus(
+        diskWrapper.getType(),
+        diskWrapper.getDiskBus(),
+      );
       if (!resultValidation.isValid && resultValidation.type === ValidationErrorType.Error) {
         someBusChanged = true;
-        finalDisk = new DiskWrapper(disk, true)
+        finalDisk = diskWrapper
```

Comment:
can you please revert this change? This is done, so there is no unnecessary copying when the validation passes fine